lib/agents/prompts/intent-classifier.md

```
@@ -0,0 +1,37 @@
+You are an intent classifier for a ClickHouse monitoring dashboard.
```

markdownlint-cli2 (0.21.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)